dali/test/python/test_operator_reader_shuffling.py

```diff
@@ -141,26 +196,52 @@ def gather_ids(pipes, epochs_run=0, batch_size=1, num_gpus_arg=None, gpus_arg=No
 def test_global_shuffle_random_shuffle():
     num_gpus = 2
     batch_size = 1
-    pipes = [COCOReaderPipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
-                                                    data_paths=data_sets[0], random_shuffle=False, stick_to_shard=False,
-                                                    shuffle_after_epoch=True, pad_last_batch=False) for gpu in range(num_gpus)]
-    [pipe.build() for pipe in pipes]
+    pipes = [
+        COCOReaderPipeline(batch_size=batch_size,
+                           num_threads=4,
+                           shard_id=gpu,
+                           num_gpus=num_gpus,
+                           data_paths=datasets[0],
+                           random_shuffle=False,
+                           stick_to_shard=False,
+                           shuffle_after_epoch=True,
+                           pad_last_batch=False)
+        for gpu in range(num_gpus)
+    ]
+
+    for pipe in pipes:
+        pipe.build()
 
     _, img_ids_list_set, _ = gather_ids(pipes)
     _, img_ids_list_set_new, _ = gather_ids(pipes)
 
     assert img_ids_list_set[0] != img_ids_list_set_new[0]
+
     assert img_ids_list_set[1] != img_ids_list_set_new[1]
-    assert img_ids_list_set[0].union(img_ids_list_set[1]) == img_ids_list_set_new[0].union(img_ids_list_set_new[1])
+
+    assert (img_ids_list_set[0].union(img_ids_list_set[1]) ==
+            img_ids_list_set_new[0].union(img_ids_list_set_new[1]))
 
 
 def test_global_shuffle_random_shuffle_2():
     num_gpus = 1
     batch_size = 1
-    pipes = [COCOReaderPipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=2,
-                                                    data_paths=data_sets[0], random_shuffle=False, stick_to_shard=False,
-                                                    shuffle_after_epoch=True, pad_last_batch=False, initial_fill=1) for gpu in range(num_gpus)]
-    [pipe.build() for pipe in pipes]
+    pipes = [
+        COCOReaderPipeline(batch_size=batch_size,
+                           num_threads=4,
+                           shard_id=gpu,
+                           num_gpus=2,
+                           data_paths=datasets[0],
+                           random_shuffle=False,
+                           stick_to_shard=False,
+                           shuffle_after_epoch=True,
+                           pad_last_batch=False,
+                           initial_fill=1)
+        for gpu in range(num_gpus)
+    ]
```

Comment:
```suggestion
        for gpu in range(num_gpus)]
```
I think most pep8 autoformatters would stick to this.

Reply:
They wouldn't if the list contents start in a new line.